plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -228,17 +228,21 @@ func (c *MaxPDVolumeCountChecker) filterVolumes(volumes []v1.Volume, namespace s
 				return nil
 			}
 
-			if pvc == nil {
-				return fmt.Errorf("PersistentVolumeClaim not found: %q", pvcName)
-			}
-
 			pvName := pvc.Spec.VolumeName
 			if pvName == "" {
-				return fmt.Errorf("PersistentVolumeClaim is not bound: %q", pvcName)
+				// PVC is not bound. It was either deleted and created again or
+				// it was forcefuly unbound by admin. The pod can still use the
+				// original PV where it was bound to -> log the error and count
+				// the PV towards the PV limit
+				glog.V(4).Infof("PVC %s/%s is not bound, assuming PVC matches predicate when counting limits", namespace, pvcName)
+				source := rand.NewSource(time.Now().UnixNano())
+				generatedID := "unboundPVC" + strconv.Itoa(rand.New(source).Intn(1000000))
```

Comment:
May be it's better to use exiting util for rand name here?

Reply:
This code is copied in all error paths in `filterVolumes`, I don't wan't a different behavior in one of them.